go.mod

```diff
@@ -5,32 +5,36 @@ go 1.21
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/jackc/pgx/v4 v4.18.2
 	github.com/jmoiron/sqlx v1.3.5
 	github.com/onsi/gomega v1.27.10
 	github.com/pkg/errors v0.9.1
 )
 
-require github.com/onsi/ginkgo/v2 v2.13.0
+require (
+	github.com/jackc/pgx/v4 v4.12.1-0.20210724153913-640aa07df17c
```

Comment:
So, we decide to use an older version finally?

Reply:
Apologies, for this confusion. 
We intended to use pgx v5. While creating the new branch for the PR, one earlier commit was mistakenly left. I've now updated the code with the correct changes. I have verified, the BD security scan, with this update, the  targeted vulnerabilities are removed.